modules/Stories/Pagination.tsx

```diff
@@ -0,0 +1,60 @@
+import type { FunctionComponent } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { PaginationProps } from 'types';
+
+const Pagination: FunctionComponent<PaginationProps> = ({ itemsTotal, currentPage, pageSize }) => {
+    const { pathname, query } = useRouter();
+    const totalPages = Math.ceil(itemsTotal / pageSize);
+
+    if (totalPages <= 1) {
+        return null;
+    }
+
+    const pagesArray = Array.from({ length: totalPages }, (_, i) => i + 1);
```

Comment:
> if I'm suggesting to overdo it.

On second thought, I would not want the page to freeze (which can happen when rendering thousands of buttons - that's why virtualization has been invented) when I open the project because it can lead to a bad first impression.

Let's improve this, we can go with something very simple (e.g.: first, previous, -3, -2, -1, current, +1, +2, +3, next, last)

Reply:
I was thinking about it, but the implementation needs a bit more thought (couldn't hack anything reasonable in short time), and initially it didn't feel like the scope of a starter.
I'll look into it again tomorrow, totally agree that it'd be nicer!

Reply:
Please have a look at this: https://github.com/prezly/prezly/tree/ce6af1de9290030dc1038e912193caa83971f2af/apps/backend/resources/javascripts/app/components/Pagination

It's an old pagination component implementation that we've had in Prezly. I'd say it's just what we need here :)

Reply:
Thanks, utilized that, and also added the gaps like in the second example you provided:
![image](https://user-images.githubusercontent.com/1641218/116360810-f810fe00-a808-11eb-832e-fc8630c33791.png)